src/App.tsx

```diff
@@ -80,19 +129,24 @@ export const App = ({ shortUrl, user }: Props) => {
     setLoading(true);
     getDiscussion(shortUrl, filters).then(json => {
       setLoading(false);
-      setComments(json.discussion.comments);
-      setPages(json.pages);
+      setComments(json?.discussion?.comments);
```

Comment:
I am going to assume if this is here that this feature works? I know my IDE always complains about it so I am never sure

Reply:
It fixed the bug I got when I tried to read the value when it was undefined so it works in that sense!

Reply:
That said ^, I'll check we don't have an issue when integrating with DCR